bliss/conf/base_config.yaml

```diff
@@ -104,6 +104,13 @@ encoder:
           - _target_: bliss.encoder.metrics.FluxError
             survey_bands: ${encoder.survey_bands}
           - _target_: bliss.encoder.metrics.GalaxyShapeError
+    plots:
+        _target_: torchmetrics.MetricCollection
+        metrics:
+          - _target_: bliss.encoder.plots.PlotSampleImages
+            frequency: 1
+            restrict_batch: 0
```

Comment:
considered this, but figured we may want to keep them separate for readability as they do serve different functions - i.e. PlotSampleImages is decidedly not a metric. 
And the plots have different functions applied to them in the encoder as well v.a.v. update, etc. which would make it difficult to merge the lists together (self.plots.update, self.report with custom plots text, etc.)
Happy to merge them together and then include a flag in the class def. but then would have to filter by that flag in the encoder anyways to apply the relevant methods, so this may indeed be easier, but will do either way

Reply:
They may start to look more similar if things like `min_flux_threshold` and `tiles_to_crop` are passed to the constructor rather than to `update(...)`

Reply:
That seems better than calling one group of metrics "plots" and another "metrics". Because some of the latter do have plotting methods. And all of the former are literally metrics (in terms of their type).

Reply:
perhaps a better name for the plots is in order; given that they do different things, I feel that they should be enumerated separately; combining the two would have, say, RMSE and PlotImages in the same list, which (other than sharing a base class) don't have much similarity in terms of true functionality... 

I will make the filtering change if you feel that's optimal

Reply:
addressed by 60375523cbc3da564a48304c723568d91f071b94